x-pack/legacy/plugins/siem/cypress/integration/lib/login/helpers.ts

```diff
@@ -39,7 +39,7 @@ const ELASTICSEARCH_PASSWORD = 'ELASTICSEARCH_PASSWORD';
 /**
  * The Kibana server endpoint used for authentication
  */
-const LOGIN_API_ENDPOINT = '/api/security/v1/login';
+const LOGIN_API_ENDPOINT = '/internal/security/login';
```

Comment:
__question:__ This code looks a bit suspicious to me, I'd love to hear what it's doing and why?